packages/orbit-components/src/Table/index.tsx

```diff
@@ -64,6 +64,8 @@ const Table = ({
         className={cx("w-full", shadows && "overflow-x-auto")}
         onScroll={handleScroll}
         ref={inner}
+        // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+        tabIndex={shadows ? 0 : undefined}
```

Comment:
This is "fixing" current violation "elements that have scrollable content are accessible by keyboard" but ideally we should also add role and aria-label to this container element to ensure accessibility. Agreed with Marco to start a new task for this https://kiwicom.atlassian.net/browse/FEPLT-2340

Reply:
I wonder if we could add role="grid" for the time being and leave the aria-label for the future. Would it make sense?

Reply:
Maybe role="region" could be used here? I'll do more research.

Reply:
I found these articles where they use role="region" in this case
https://www.tempertemper.net/blog/accessible-responsive-tables
https://adrianroselli.com/2017/11/a-responsive-accessible-table.html#ResponsiveScrollingKeyboard

And also in MDN if I understand correctly https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role#scrolling_content_areas_with_overflow_text

But once there is a role, there should be a label as well 😬

Reply:
Yeah, I've tried the region role but it's not an interactive role so the warning is still there.
Even with role="region" + aria-label I'm not able to get rid of the warning :D

Reply:
I'll resolve this. To be tackled as part of the Jira ticket you've created.

Reply:
At least the aria-label works well on the screen reader 😅 

<img width="587" alt="image" src="https://github.com/user-attachments/assets/388f0b3e-dc08-49b4-ae83-8d126609aef5" />